### PR TITLE
create LICENCE

### LINKED ISSUE #129: Add LICENCE
Hey, @anmode   I'm active GSSoC'23 contributor and I would like to add a license file to this repository. Please assign me to this issue. Thank you!

### PR DESCRIPTION
**Pull request by Jiggy9**
closes #129

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 anmode
```
Comment (anmode) [inline LICENCE:3]: I do not think it is correct! please see other repo or example how to create a license...and this is open sourced startup project. So you have to think this is mind. thanks

**Review by anmode — CHANGES_REQUESTED** [review]
Some changes

File: LICENCE (hunk 1 above)
Comment (Jiggy9) [reply LICENCE:3]: yup now I understand I'll send new PR, thank you!

Comment (anmode) [reply LICENCE:3]: You do not need to create a New pR!

Comment (anmode) [reply LICENCE:3]: you can continue in this one! too

**Pull request closed without merging**